modules/platforms/cpp/ignite/common/ignite_type.h

```diff
@@ -70,6 +74,7 @@ enum class column_type {
 
     /** Binary data. */
     BYTE_ARRAY = 15,
+    BINARY = BYTE_ARRAY,
```

Comment:
Do you think we need this synonym?

Reply:
I kind of agree, but this is a common name across all Ignite codebase.

Reply:
Removed